backend/src/handlers/proxy_handler.go

```diff
@@ -64,6 +64,11 @@ func (srv *Server) handleForwardKiwixProxy(w http.ResponseWriter, r *http.Reques
 			},
 		},
 		ModifyResponse: func(res *http.Response) error {
+			//MAKE SURE TO NOT CACHE MAIN HTML CONTENT | WE MAY NEED TO ADD OTHER CONTENT TYPE TO NOT CACHE
+			contentType := res.Header.Get("Content-Type")
+			if contentType == "text/html" || contentType == "" {
+				res.Header.Set("Cache-Control", "no-store, no-cache, must-revalidate, private")
```

Comment:
Might want to also set `Pragma: no-cache` as well

Reply:
added http/1.0 Pragma no-cache header